csharp/src/Ice/ReplyStatus.cs

```diff
@@ -9,44 +9,33 @@ namespace Ice;
 /// </summary>
 public enum ReplyStatus : byte
 {
-    /// <summary>
-    /// A successful reply message.
-    /// </summary>
+    /// <summary>The dispatch completed successfully.</summary>
     Ok = 0,
 
-    /// <summary>
-    /// A user exception reply message.
-    /// </summary>
+    /// <summary>The dispatch completed with a Slice user exception.</summary>
     UserException = 1,
 
-    /// <summary>
-    /// The target object does not exist.
-    /// </summary>
+    /// <summary>The dispatch could not find an implementation for the target object.</summary>
     ObjectNotExist = 2,
 
-    /// <summary>
-    /// The target object does not support the facet.
+    /// <summary>The dispatch found an implementation for the target object but could not find the requested facet.
     /// </summary>
     FacetNotExist = 3,
 
-    /// <summary>
-    /// The target object does not support the operation.
+    /// <summary>The dispatch found an implementation for the target object but could not find the requested operation.
     /// </summary>
     OperationNotExist = 4,
 
-    /// <summary>
-    /// The dispatch failed with an Ice local exception.
-    /// </summary>
+    /// <summary>The dispatch failed with an Ice local exception.</summary>
     UnknownLocalException = 5,
 
-    /// <summary>
-    /// The dispatch failed with a Slice user exception that does not conform to the exception specification of the
-    /// operation.
-    /// </summary>
+    /// <summary>The dispatch failed with a Slice user exception that does not conform to the exception specification of
+    /// the operation.</summary>
     UnknownUserException = 6,
 
-    /// <summary>
-    /// The dispatch failed with some other exception.
-    /// </summary>
-    UnknownException = 7
+    /// <summary>The dispatch failed with some other exception.</summary>
+    UnknownException = 7,
+
+    /// <summary>The caller is not authorized to access the requested resource.</summary>
```

Comment:
See also: 
https://github.com/icerpc/icerpc-slice/blob/main/IceRpc/StatusCode.slice

DeadlineExceeded is not meaningful at present since we don't sent any deadline with request.
Unavailable maps unfortunately to ObjectNotExist. 
TruncatedPayload is not meaningful with Ice.

InvalidData requires a bit of work to do properly... maybe for 3.9.